app/controllers/moves_controller.rb

```diff
@@ -0,0 +1,29 @@
+class MovesController < ApplicationController
+  def create
+    # TODO: check the move and any piece to be captured.
+    # now assume the move is valid and the target square is empty,
+    # move the piece and deselect it.
+    pieces_selected = Piece.where(is_selected: true)
```

Comment:
Line six will grab every piece in the database where is_selected is equal to true.  If there is more than one game going on at a time this could cause big issues. Might want to scope this a little more strictly.